tailwind.config.ts

```diff
@@ -18,6 +18,7 @@ const config: Config = {
         'hover-1': '#4db6ac', // TODO: Decide on a color for this
         'hover-2': '#89B4FA', // TODO: Decide on a color for this
         'event-banner': '#5E18EB', // Event Banner Background
+        socials: '#7BD4CB', // Social
```

Comment:
Also is this the colour we're using on the social buttons on the hero banner right now? I'll send a Discord message about it

Reply:
> Is there a functional/semantic reason socials isn't in quotes and the rest are?

The others have a `-` in their name which means they have to be in quotes the formatter removes them on this one because they are not needed. We will probably change this name with issue #16